src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php

```diff
@@ -57,6 +57,14 @@ public function iExpectAConfirmationDialogShouldAppear()
     }
 
     /**
+     * @Given I click the back icon
+     */
+    public function iClickOnTheBackIcon()
+    {
+        $this->clickSelector('.fa-chevron-left');
```

Comment:
it is used but on the page will redirect no events will work. but i use it to check the save (see feature files).